app/upgrades.go

```diff
@@ -96,7 +96,13 @@ var handlers = map[string]appUpgrade{
 		}, // upgrade for pio-testnet-1 from v1.8.0-rc8 to v1.8.0-rc9
 	},
 	"kahlua": {}, // upgrade for pio-testnet-1 from v1.8.0-rc9 to v1.8.0
-	"lava":   {}, // 1.9.0
+	"lava": {
+		Handler: func(app *App, ctx sdk.Context, plan upgradetypes.Plan) (module.VersionMap, error) {
+			app.MarkerKeeper.RemoveSendEnabledForMissingMarkers(ctx)
+			versionMap := app.UpgradeKeeper.GetModuleVersionMap(ctx)
+			return versionMap, nil
+		}, // upgrade for 1.9.1
```

Comment:
might not be 1.9.1 anymore ... but we can tweak this in the release branch this ends up in.

Reply:
yep, updated to 1.10.0